src/omlt/block.py

```diff
@@ -1,236 +1,66 @@
 import warnings
+import weakref
```

Comment:
These no longer get used in this module. Should be okay to remove?

Reply:
fixed